Automattic/wp-calypso
# State: Remove unused site keyrings save requests state

#### Changes proposed in this Pull Request

This PR removes some site keyrings save requests state that is not in use. 

#### Testing instructions

* Verify Calypso builds without errors.
* Smoke test connections under `/marketing/connections/site` and verify they still work well (retrieving, adding, and removing)
* Verify that removed code is not in use.
